opensearch-observability/src/test/kotlin/org/opensearch/observability/PluginRestTestCase.kt

```diff
@@ -63,9 +63,12 @@ abstract class PluginRestTestCase : OpenSearchRestTestCase() {
         return true
     }
 
+    open fun preserveOpenSearchIndicesAfterTest(): Boolean = false
+
     @Throws(IOException::class)
     @After
     open fun wipeAllOpenSearchIndices() {
+        if (preserveOpenSearchIndicesAfterTest()) return
```

Comment:
If this is set to false at 66, when will it be true??

Reply:
It's set to True in BWC IT, so whenever bwc IT is triggered this is triggered too

Reply:
Ohp just kidding, I found it in the next file!